test/promise-component-mocha.js

```diff
@@ -118,6 +118,32 @@ describe('promise-component', function() {
             return onceData(component, 'outgoing', 'sentback', 'incoming', "hello");
         }).should.be.rejectedWith("hello world");
     });
+    it("should log rejections to console.error when no reject port attached", function() {
+        return Promise.resolve({
+            inPorts:{
+                'in':{
+                    ondata: function(payload) {
+                        throw payload + " world";
+                    }
+                }
+            }
+        }).then(promiseComponent).then(createComponent).then(function(component){
+            var console_error = console.error;
+            return new Promise(function(resolve, reject) {
+                console.error = reject;
```

Comment:
Also do not follow how this really tests console logging.  Perhaps we should go over this on Monday.